codespell_lib/data/dictionary_names.txt

```diff
@@ -6,5 +6,3 @@ liszt->list
 que->queue
 sargent->sergeant, argent,
 tim->time
-wen->when
-wight->weight, white, right, write,
```

Comment:
I pointed out Isle of Wight which I think this went in names.

See also these people:
https://en.wikipedia.org/wiki/Wight_(surname)

Perhaps we should split names into first and surname in terms of likelihood of clashes.